codebuild/spec/buildspec_omnibus.yml

```diff
@@ -0,0 +1,239 @@
+          S2N_LIBCRYPTO: openssl-1.1.1
+          BUILD_S2N: true
+          TESTS: integration
+          GCC_VERSION: 6
+          S2N_COVERAGE: true
+          CODECOV_IO_UPLOAD: true
+
+    # Saw
+    - identifier: s2nSawBike
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_2XLARGE
+        variables:
+          S2N_LIBCRYPTO: openssl-1.0.2
+          TESTS: sawBIKE
+          SAW: true
+          GCC_VERSION: NONE
+
+    - identifier: sawHMACPlus
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TESTS: sawHMACPlus
+          SAW: true
+          GCC_VERSION: NONE
+
+    - identifier: s2nSawTls
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TESTS: tls
+          SAW: true
+          GCC_VERSION: NONE
+
+    # Other
+    - identifier: s2nSidetrail
+      buildspec: codebuild/spec/buildspec_sidetrail.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_2XLARGE
+        image: 024603541914.dkr.ecr.us-west-2.amazonaws.com/linux-docker-images:sidetrail
+        variables:
+          TESTS: sidetrail
+
+
+    - identifier: s2nValgrindOpenSSL102Gcc6Fips
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TESTS: valgrind
+          GCC_VERSION: 6
+          S2N_LIBCRYPTO: openssl-1.0.2-fips
+          BUILD_S2N: true
+
+    - identifier: s2nValgrindOpenSSL111Gcc9
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TESTS: valgrind
+          GCC_VERSION: 9
+          S2N_LIBCRYPTO: openssl-1.1.1
+          BUILD_S2N: true
+
+    - identifier: s2nvalgrind_openssl102
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TESTS: valgrind
+          GCC_VERSION: '6'
+          S2N_LIBCRYPTO: 'openssl-1.0.2'
+          BUILD_S2N: 'true'
+
+    - identifier: s2nAsanOpenSSL111Coverage
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_SMALL
+        variables:
+          TESTS: asan
+          GCC_VERSION: '6'
+          S2N_LIBCRYPTO: 'openssl-1.1.1'
+          BUILD_S2N: 'true'
+          S2N_COVERAGE: 'false'
+          CODECOV_IO_UPLOAD: 'false'
+          CODECOV_TOKEN: 'foobar'
+
+    - identifier: s2nAsanOpenssl102
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_SMALL
```

Comment:
These two are ok to be small?

Reply:
Correct, example [here](https://us-west-2.console.aws.amazon.com/codesuite/codebuild/024603541914/projects/s2nOmnibus/build/s2nOmnibus%3Ac02d2ae6-c867-432a-9f7b-6565ac4a4a73?region=us-west-2)